quantmsrescore/ms2pip.py

```
@@ -105,8 +109,8 @@ def process_spectra(
 
         # Add XGBoost predictions if required
         if (
-            not (vector_file or annotations_only)
-            and "xgboost_model_files" in MODELS[self.model].keys()
+                not (vector_file or annotations_only)
+                and "xgboost_model_files" in MODELS[self.model].keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)